File: Documentação/Documentação/Template Latex - Relatorio Tecnico - IFSP - SBV/fontes/LibGDX/move.java
public void move(Vector2 direction) {
    Vector2 newPosition = new Vector2(position.x + direction.x, position.y + direction.y);
    if (!isValidMove(newPosition, blocks)) {
        return;
    }
    position.x += direction.x;
    position.y += direction.y;
    tetris.lockTimer = 0;
}

File: Documentação/Documentação/Template Latex - Relatorio Tecnico - IFSP - SBV/fontes/Raylib/move.cpp
void Piece::move(Vector2 direction){
    if(pointerToTetris == nullptr){
        return;
    }
    Vector2 newPosition = {position.x + direction.x, position.y + direction.y};
    if(isValidMove(direction, blocks)){
        position = newPosition;
    }
}